src/content/docs/en/guides/authentication.mdx

```diff
@@ -69,7 +69,7 @@ To install `auth-astro` manually, install the required package for your package
 
 Then, apply the integration to your `astro.config.*` file using the `integrations` property:
 
-```ts title="astro.config.mjs" ins={2,6}
+```ts title="astro.config.ts" ins={2,6}
```

Comment:
Hey @Kevin2606, any reason the Astro config file was also changed to `.ts`? From my understanding only the `auth.config.*` file would need to be in TypeScript for it to be fixed? I am no expert in Auth.js, so let me know if I am wrong here!

Reply:
Hey @yanthomasdev, thanks for pointing that out! That change was actually a mistake on my part. The `astro.config` file doesn't need to be changed to `.ts`, and there was no reason to modify its extension. It should have remained as `astro.config.mjs`. I’ll revert that change right away.

Reply:
Oh, I missed it in my review. Thanks, Yan for catching it.